source/server/configuration_impl.cc

```diff
@@ -92,40 +88,49 @@ void MainImpl::initialize(const Json::Object& json) {
   }
 }
 
-void MainImpl::initializeTracers(const Json::Object& tracing_configuration) {
+void MainImpl::initializeTracers(const Json::Object& configuration) {
   log().info("loading tracing configuration");
 
-  // Initialize tracing driver.
-  if (tracing_configuration.hasObject("http")) {
-    Json::ObjectPtr http_tracer_config = tracing_configuration.getObject("http");
-    Json::ObjectPtr driver = http_tracer_config->getObject("driver");
+  if (!configuration.hasObject("tracing")) {
+    http_tracer_.reset(new Tracing::HttpNullTracer());
+    return;
+  }
 
-    std::string type = driver->getString("type");
-    log().info(fmt::format("  loading tracing driver: {}", type));
+  Json::ObjectPtr tracing_configuration = configuration.getObject("tracing");
+  if (!tracing_configuration->hasObject("http")) {
```

Comment:
Can you check to make sure we have test coverage on both of these branches? It might be better to combine this into the previous if check for easier coverage, but as long as we have coverage of both I'm fine with either way.